docs/policygenerator-reference.yaml

```
@@ -21,6 +21,10 @@ policyDefaults:
   # annotation. This defaults to ["CM-2 Baseline Configuration"].
   controls:
     - "CM-2 Baseline Configuration"
+  # Optional. Default value true will consolidate multi manifests 
+  # specified in policyConf into single policy template. 
+  # Set to false will skip this consolidation.
```

Since this is user facing documentation, I think we should drop the usage of `policyConf` here since it's a Go type in our project and not known to the user. How about something like this?

```suggestion
  # Optional. This determines if a single configuration policy should be
  # generated for all the manifests being wrapped in the policy.
  # If set to false, a configuration policy per manifest will be generated.
  # This defaults to true.
```